src/user/interfaces/user.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { Document } from 'mongoose';
```

Comment:
We will be using nestjs/mongoose and planning to remove `mongoose` completely.

Reply:
sure

Reply:
@Abhishek-kumar09 we can't remove mongoose completely.. because @nestjs/mongoose is just a superset of it and for using document we would need mongoose